### DISCUSSION TITLE
planning meetings

### DISCUSSION THREAD

**Discussion by RLC-DCPPC** [IGS/gEAR#172] [Ideas] [answered]
10/5
 - demo of Carlo's brain dev thing - Brian
 - integrating 3D Umap display
 - projectR interim impl - Carlo

1/26 
- Joe on testing outcomes; 
- Kevin; 
- Fun Retro (https://easyretro.io/publicboard/YRqL6M2JyldoRnjoRyEIGNs8z0x2/cb6f3fa9-02e0-4d40-9f9b-9db912dc1655)

2/9  

- Suggestions for gEAR front page
- Brian

2/23

- ?

Topic Backlog

- Testing [tips](https://www.thirdandgrove.com/insights/tips-on-testing-in-an-agile-environment-and-pitfalls-to-avoid/) -

**Comment by adkinsrs** [comment, marked as answer]
My two cents.  I feel tickets of this type should be moved to the "Discussions" tab since they don't really highlight any issue or tasks related to the codebase itself.  Thoughts?

**Reply by RLC-DCPPC** [reply]
Good idea. Done.

**Comment by jorvis** [comment]
Agreed.

**Comment by adkinsrs** [comment]
Same with #158 and #159

**Comment by adkinsrs** [comment]
I think I will talk briefly on improvements on #150 for the standup on 1/19
